gui/cppchecklibrarydata.h

```diff
@@ -171,13 +171,19 @@ class CppcheckLibraryData {
         QString sign;
     };
 
+    struct TypeChecks {
```

Comment:
I am not sure if we want to have a struct here. How about a type:

    using TypeChecks = QList<QPair<QString, QString>>;

?

Reply:
My intention was to keep multiple `type-checks` sections separated and write it that way back to file. But i think it's ok to have a single list for all `type-checks`.
I will change it.

Reply:
Ok, first i was on the wrong way but i think finally and late at night i understood your comment.

There is one CI integration test failing but i didn't find any hint within log that it's because of my last commit.
Is there a way to run tests again ?